ci-operator/step-registry/hypershift/kubevirt/create/hypershift-kubevirt-create-commands.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+HCP_CLI="/usr/bin/hcp"
+
+MCE=${MCE_VERSION:-""}
+CLUSTER_NAME=$(echo -n "${PROW_JOB_ID}"|sha256sum|cut -c-20)
+if [[ -n ${MCE} ]] ; then
+    CLUSTER_NAMESPACE_PREFIX=local-cluster
+else
+    CLUSTER_NAMESPACE_PREFIX=clusters
+fi
+#CLUSTER_NAMESPACE=${CLUSTER_NAMESPACE_PREFIX}-${CLUSTER_NAME}
```

Comment:
I think we can safely remove it.
I got it from https://github.com/openshift/release/blob/master/ci-operator/step-registry/hypershift/mce/kubevirt/create/hypershift-mce-kubevirt-create-chain.yaml#L41 but neither there seems to be really in use.
Shellcheck is not failing tehre on it since we have there a bash script embedded in a yaml file and so it's not checked.